torch/_inductor/codecache.py

```diff
@@ -2006,7 +2007,8 @@ def _pad_to_alignment(raw_bytes: bytes) -> bytes:
                     compile_file(input_path, output_o, compile_cmd.split())
                     os.chmod(output_o, 0o644)
                 else:
-                    run_command_and_check(compile_cmd)
+                    with dynamo_timed("run_compile_cmd", log_pt2_compile_event=True):
```

Comment:
Didn't you already add dynamo_timed to run_command_and_check?

Reply:
I think I added them so I can annotate which run_command_and_check it is. But let me think about it more next week. 

<img width="1235" alt="Screenshot 2024-11-12 at 2 44 44 PM" src="https://github.com/user-attachments/assets/648b766f-4949-4a36-b166-b43285b6d28f">